### DISCUSSION TITLE
Captcha for SignUp / Login

### DISCUSSION THREAD

**Discussion by monisharajan** [ravindermahajan890/tamashflix-ms-clone#33] [Ideas]
Lets discuss about the different methods of captchas available
Reason:
1. Anybody on internet should not exploit our API (DDOS)
2. To avoid getting charged unnecessarily because of so many account created

**Comment by as-ajitsingh** [comment]
Possible Options: 
1. Old school image captcha with 5-6 chars
2. Google Captcha
